File: 1. Two Sum/solution.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        for first in 0..<nums.endIndex {
            let minus = target - nums[first]
            for second in (first + 1)..<nums.endIndex {
                if nums[second] == minus {
                    return [first, second]
                }
            }
        }
        return []
    }
}

File: 1. Two Sum/solution.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target) {
    for (int *p = nums; p < nums + numsSize; p ++) {
        int minus = target - *p;
        for (int *nextP = p + 1; nextP < nums + numsSize; nextP ++) {
            if (*nextP == minus) {
                int *answer = malloc(sizeof(int) * 2);
                *(answer) = (int)(p - nums);
                *(answer + 1) = (int)(nextP - nums);
                return answer;
            }
        }
    }
    return NULL;
}